python310Packages.appthreat-vulnerability-db: 4.0.4 -> 4.1.6

fabaff opened the pull request:
###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python310Packages.google-reauth: init at 0.1.1

fabaff committed:
python310Packages.google-apitools: init at 0.5.32

fabaff committed:
python310Packages.appthreat-vulnerability-db: 4.0.4 -> 4.1.5

Diff: https://github.com/AppThreat/vulnerability-db/compare/refs/tags/v4.0.4...v4.1.5

Changelog: https://github.com/AppThreat/vulnerability-db/releases/tag/v4.1.5

SuperSandro2000 commented on pkgs/tools/security/cve-bin-tool/default.nix:
```diff
@@ -69,18 +68,17 @@ buildPythonApplication rec {
     pytestCheckHook
   ];
 
+  # Wants to open a sqlite database, access the internet, etc
```
A sqlite database can be created with touch.

fabaff replied:
Looks like that `cve-bin-tool` need a do over anyways.

fabaff committed:
python310Packages.appthreat-vulnerability-db: 4.1.5 -> 4.1.6

Diff: https://github.com/AppThreat/vulnerability-db/compare/refs/tags/v4.1.5...v4.1.6

Changelog: https://github.com/AppThreat/vulnerability-db/releases/tag/v4.1.6

fabaff commented:
Result of `nixpkgs-review pr 204139` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>appthreat-depscan</li>
    <li>python310Packages.appthreat-vulnerability-db</li>
    <li>python39Packages.appthreat-vulnerability-db</li>
  </ul>
</details>

marsam commented:
@GrahamcOfBorg eval
